app/packages/core/src/components/Modal/Modal.tsx

```diff
@@ -156,7 +156,8 @@ const Modal = () => {
             return newSelected;
           });
         } else if (e.key === "Escape") {
-          if (activeLookerRef.current) {
+          const mediaType = await snapshot.getPromise(fos.mediaType);
```

Comment:
This is unfortunate...someday we should re-work our key bindings to prevent this logic from being added here. No action required.